index.js

```diff
@@ -1 +1,12 @@
-// TODO: Start prototyping here
+import esMain from 'es-main';
+import { host, port } from './src/config.js';// eslint-disable-line import/extensions
+import HttpServer from './src/httpServerClass.js';// eslint-disable-line import/extensions
+
+const server = new HttpServer();
+
+export default server;
+
+if (esMain(import.meta)) {
+    // Для линтера добавлен парсер babel-eslint, чтобы он не ругался на import.meta
+    server.startServer(host, port).then();
```

Comment:
We do not need `.then` here. `async` functions start execution as soon as you call them. If you have problems with lint - this place is valid exception for the `no //eslint-disable comments` rule, just suppress the error here. Since we know what we do here. However a comment can be helpful here (to show that you understand what you're doing)